docs/0.2/creating_schemas.md

```diff
@@ -80,7 +80,7 @@ Sawtooth]({% link docs/0.2/grid_on_sawtooth.md %}) for more information.)
 If you are not using this example environment, replace these items with the
 actual values for your environment when entering each command.
 
-### Connect to a Grid Node
```

Comment:
We use the imperative everywhere else in the docs so I don't think we should change that here

Reply:
Ok, I was bringing them in line with the menu and h1 headers, but we can address tense in another pr if needed.